projects/ardin_mb/visual_navigation_bob.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+// Ardin MB includes
+#include "visual_navigation_base.h"
+
+// BoB Robotics includes
+#include "navigation/perfect_memory.h"
+
+//----------------------------------------------------------------------------
+// BoBWrapper
+//----------------------------------------------------------------------------
+//! Very thin wrapper to allow VisualNavigationBase wrap BoB
+//! navigation algorithms which can be used without additional state etc
+template<typename T>
+class VisualNavigationBoB : public VisualNavigationBase
```

Comment:
So, this is inheriting from the _project-specified_ ``VisualNavigationBase`` class rather than the BoB robotics one which _is_ runtime polymorphic so, if we removed the base class from BoB robotics, this would still save typing when interfacing with simpler stateless BoB robotics algorithms which can just be used by calling ``test()``